youthful-piety/modernized/09-chapter-08.adoc

```diff
@@ -344,15 +350,22 @@ In a letter addressed to a youth with whom he had
 formed an intimate acquaintance at school,
 he says,
 
+[.embedded-content-document.letter]
+--
+
 I hope as we travel on through this transitory state,
 we shall be enabled to cast the cares of this world behind us,
 looking to the Author of all good, who will help all those who trust in Him, I believe,
 beyond all doubt, that all those who trust in Him, will not lack the bread of life.
 
+--
+
 The following extracts from a diary which he kept,
```

Comment:
From here down, there are some diary excerpts and a letter sandwiched in between. I wasn't sure if the diary stuff should still be embedded or not.